src/ORM/Association/BelongsToMany.php

```diff
@@ -270,7 +270,7 @@ public function junction($table = null): Table
 
             $config = [];
             if (!$tableLocator->exists($tableAlias)) {
-                $config = ['table' => $tableName];
+                $config = ['table' => $tableName, 'useFallbackClass' => true];
```

Comment:
Should the option be `allowFallbackClass`? Could `use` be misinterpreted, as 'always use'?

Reply:
Could then also be shorter with `allowFallback` maybe.

Reply:
Okay, I'll rename the option to `allowFallbackClass`, I prefer keeping the `Class` suffix as it clearly indicates what we are allowing :).

The method name `useFallbackClass()` is fine right?

Reply:
Hm I wonder if the method name should also match the config key
But I am fine either way.